item-spec/README.md

```diff
@@ -1,19 +1,23 @@
 # STAC Items
 
-The core of a SpatioTemporal Asset Catalog (STAC) is a set of JSON fields defined by the 
-[STAC Item spec](item-spec.md). These fields define an **Item** - the entity that contains 
-metadata for search as well as links to the actual assets that they represent. Their main function 
-is as the leaf nodes of a [Catalog](../catalog-spec/catalog-spec.md).
-See the [overview](../overview.md) document for more information on how all the pieces fit together.
+The [STAC Item spec](item-spec.md) defines the most important object in a STAC system. An
```

Comment:
Super minor, but in the other two readme's we now just start in on 'STAC [Catalog](catalog-spec.md) object' and 'STAC [Collection](collection-spec.md) object', so for consistency we could use STAC Item object, instead of STAC Item Spec. (obviously would have to change the grammar around a bit.